src/software/networking/threaded_unix_sender.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import queue
+import socket
+from threading import Thread
+
+from google.protobuf import text_format
+from google.protobuf.any_pb2 import Any
+
+
+class ThreadedUnixSender:
+    def __init__(self, unix_path, max_buffer_size=3, convert_to_any=True):
+
+        """Send protobufs over unix sockets
+
+        :param unix_path: The unix path to send the protobuf to
+        :param max_buffer_size: The size of the send buffer
+        :param convert_from_any: Convert from any
```

Comment:
doesn't match param name, and the description for it could be clearer

Reply:
fixed and added some mroe info